nolearn/lasagne/visualize.py

```diff
@@ -338,29 +347,31 @@ def make_pydot_graph(layers, output_shape=True, verbose=False):
     return pydot_graph
 
 
-def draw_to_file(layers, filename, **kwargs):
+def draw_to_file(net, filename, **kwargs):
```

Comment:
To keep backward compatibility with the code that we started out with.  If a Lasagne user wants to use Eben's code packaged up in a library, they now can.